src/components/Gallerie/Gallery.jsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Virer et appliquer les corrections nécessaires :

/* eslint-disable no-unused-vars */
/* eslint-disable indent */
/* eslint-disable guard-for-in */
/* eslint-disable no-restricted-syntax */